src/store/printer/types.ts

```diff
@@ -726,6 +726,7 @@ type KlipperPrinterMmuMachineStateBaseType = {
     filament_always_gripped: boolean;
     has_bypass: boolean;
     multi_gear: boolean;
+    environment_sensor: string;
```

Comment:
Make this optional as older versions will not have this

```suggestion
    environment_sensor?: string;
```